### ISSUE TITLE
[BUG] Markers only disappear once you re-open the file

### ISSUE THREAD

**Issue by hogejo** [Nasller/CodeGlancePro#161]
Thanks for the plugin! This is an inconvenience, not a "full on bug". But I want to help other new users running into this:

What steps will reproduce the issue?
    1. Observe markers in the minimap
    2. Disable markers in the settings
    3. Apply and close settings, but don't close the file
    4. Markers don't disappear

However, they _are disabled_. You just have to close the file (and potentially other open files too).

At the same time, if you change settings on Markers, the changes will appear without closing the file - so this can be confusing for ones who don't want markers.

Details:

```
IntelliJ IDEA 2024.3.1 (Ultimate Edition)
Build #IU-243.22562.145, built on December 9, 2024
Runtime version: 21.0.5+8-b631.28 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 6.11.8-1-default
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 3484M
Cores: 16
Non-Bundled Plugins:
  com.nasller.CodeGlancePro (1.9.4)
Kotlin: 243.22562.145-IJ
```

**Comment by Nasller** [comment]
[CodeGlancePro-1.9.5-SNAPSHOT.zip](https://github.com/user-attachments/files/18176607/CodeGlancePro-1.9.5-SNAPSHOT.zip)
please try this

**Comment by hogejo** [comment]
Works like a charm!

**Comment by hogejo** [comment]
Woops. I will leave you to decide if you want to close this now or only when you've released 1.9.5 with the fix. Thanks!
